modules/getting-started/pages/glossary.adoc

```diff
@@ -0,0 +1,203 @@
+
+= D
+dataloader.io::
+The data loader for Salesforce.
+
+DataSense::
+A feature of Anypoint Studio that uses message metadata to facilitate application design. With DataSense, Anypoint Studio proactively acquires information such as data type and structure, in order to prescribe how to accurately map or use this data in your application.
+
+DataWeave::
+The DataWeave Language queries and transforms data for integrations.
+
+= E
+
+Edge::
+Anypoint Security feature to help you manage edge security.
+
+= F
+
+flow::
+A simple, flexible mechanism that enables orchestration of services using the sophisticated message flow capabilities of Mule® runtime engine.   Also referred to as service orchestration.
+
+= H
+
+hybrid deployment::
+Deploying a Mule Application via the cloud console of the Anypoint Runtime Manager to an on-premises server that hosts Mule runtime engine. This deployment is hybrid in the sense that the hosting of your application is on-premises, while application management is in the cloud.
+
+= M
+message filter::
+A message processor that controls whether a message is processed by a filter.
+
+message processor::
+A basic building block used to construct flows. A message processor controls how messages are sent and received within a flow. 
+
+message receiver::
+A Java class used by a connector to read the incoming data, package it as a message, and passes it to a service component’s inbound router. The message receiver can use a transformer if necessary to convert the data.
+
+mocking service::
+Provides a public URI to an API being designed. Exercising the URI returns the responses defined in the API specification for testing or exploring an API.
+
+Mule® application::
+An app that is configured to run in Mule runtime engine.
+
+module::
+A construct that groups together components. Modules add flexibility to your apps by allowing you to aggregate values, compress data, use Java features, and use extra features for processing JSON.
+
+Mule Enterprise Edition (EE)::
+The enterprise version of Mule, available for 30-day trial download. The Enterprise Edition includes full development cycles, testing, technical support, maintenance releases and hot fixes, and management and monitoring tools from MuleSoft. If you are deploying Mule in a mission-critical environment, want to ensure that you always have a stable, high-quality release, and want additional tools for managing and monitoring your deployment, you should purchase a subscription to Mule Enterprise Edition.
+
+Mule® runtime engine (Mule® or Mule)::
+Java-based integration runtime engine for Anypoint Platform. uses a staged event-driven architecture (SEDA) to enqueue messages and process them inside of flows in separate stages. Mule is the engine for Anypoint Platform™ and the only runtime that combines data and application integration across legacy systems, SaaS applications, and APIs.
```

Comment:
Is the second sentence meant to begin with the word "uses?" If so, uppercase.

Reply:
I deleted everything after the first sentence to conform with other entries.